### PR TITLE
Update README.md with YourStop

### PR DESCRIPTION
**Pull request by harringtonp**
Add YourStop to Web Apps list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ Apps people use when taking transit.
 - [Brand New Subway](http://jpwright.net/subway/) - An interactive transportation planning game that lets players alter the NYC subway system to their heart's content.
 - [CityMapper Webapp](https://citymapper.com/nyc) - Really polished webapp with trip planner and route status for over 30 of cities.
 - [Google I/O Transport Tracker](https://github.com/googlemaps/transport-tracker) - Shows shuttle arrival times for Google I/O conference, based on the open-source [transport-tracker project](https://github.com/googlemaps/transport-tracker).  Note: To implement this yourself, you need a [Google Maps APIs Premium Plan license](https://developers.google.com/maps/pricing-and-plans/).
+- [YourStop](http://yourstop.info) - Easy to use and mobile friendly web app which consumes GTFS feeds and displays both live and scheduled trips for stops. Launched with MBTA, YRT/Viva and Maryland MTA.
```
Comment (barbeau) [inline README.md]: @harringtonp Could you please remove the "Easy to use and" part of this?  I like to keep this list as objective as possible.

**Review by barbeau — CHANGES_REQUESTED** [review]
@harringtonp Thanks!  One small change requested inline.

**Review by barbeau — APPROVED** [review]

**Comment by barbeau** [comment]
Thanks @harringtonp, looks good!

**Comment by harringtonp** [comment]
Thanks Sean, that's my first  gihub pull request done

**Pull request merged**